scripts/benchmark-rule.js

```diff
@@ -26,6 +27,7 @@ if (
 ) {
   parsedOptions = JSON.parse(ruleOptions)
 }
+/* eslint-disable eqeqeq */
```

Comment:
p.s. This inline comment doesn't show the `/* eslint-disable eqeqeq */` a couple of lines up at line 21